### PR TITLE
Docs: Improvements and minor fixes in WBEMSubscriptionManager section

### PR DESCRIPTION
**Pull request by andy-maier**
Ready for review and merge.

### CODE REVIEW

File: pywbem/_subscription_manager.py [hunk 1]
```diff
@@ -779,11 +781,14 @@ def add_filter(self, server_id, source_namespace, query,
 
     def get_owned_filters(self, server_id):
         """
-        Return the owned indication filters in a WBEM server that have been
-        created by this subscription manager.
+        Return the indication filters in a WBEM server that are owned by this
```
Comment (KSchopmeyer) [inline pywbem/_subscription_manager.py]: You can drop the words 'that are'

File: pywbem/_subscription_manager.py [hunk 2]
```diff
@@ -779,11 +781,14 @@ def add_filter(self, server_id, source_namespace, query,
...
+        subscription manager.
 
         This function accesses only the local list of owned filters; it does
-        not contact the WBEM server.
+        not contact the WBEM server. The local list of owned filters is
+        discovered from the WBEM server when the server is registered with the
+        subscription manager, and is from then on maintained as changes happen
```
Comment (KSchopmeyer) [inline pywbem/_subscription_manager.py]: change "is from then on maintained as" to is maintained from then on"

File: pywbem/_subscription_manager.py [hunk 3]
```diff
@@ -990,11 +995,14 @@ def add_subscriptions(self, server_id, filter_path, destination_paths=None,
...
-        Return the owned indication subscriptions for a WBEM server that have
-        been created by this subscription manager.
+        Return the indication subscriptions in a WBEM server that are owned by
+        this subscription manager.
 
         This function accesses only the local list of owned subscriptions; it
-        does not contact the WBEM server.
+        does not contact the WBEM server. The local list of owned subscriptions
+        is discovered from the WBEM server when the server is registered with
+        the subscription manager, and is from then on maintained as changes
```
Comment (KSchopmeyer) [inline pywbem/_subscription_manager.py]: Another case "is from then on maintained" to "is maintained from then on"

File: pywbem/_subscription_manager.py (hunk 2 above)
Comment (andy-maier) [reply pywbem/_subscription_manager.py]: DONE (3 occurrences).

File: pywbem/_subscription_manager.py (hunk 3 above)
Comment (andy-maier) [reply pywbem/_subscription_manager.py]: DONE

File: pywbem/_subscription_manager.py (hunk 1 above)
Comment (andy-maier) [reply pywbem/_subscription_manager.py]: DONE (4 occurrences)

**Pull request merged**